### ISSUE TITLE
virtualbox cli

### ISSUE THREAD

**Issue by danielmichaels** [tldr-pages/tldr#3068]
## Tldr command `virtualboxvm` not functioning as expected

The Virtualbox CLI doesn't function as shown in `tldr` pages. Before raising a PR I wanted to clarify if it is a issue on my end, or a global problem.

In the [tldr](https://github.com/tldr-pages/tldr/blob/master/pages/common/virtualboxvm.md), the executable for the Virtualbox's CLI is `virtualboxvm`. On my system `5.1.4-arch1-1-ARCH` using virtualbox version `6.0.8 r130347` it is `VBoxManage` or `vboxmanage`. 

Additionally, the commands in the `tldr` only pertain to starting a VM. I would like to see `VBoxManage controlvm '<vm_name>' poweroff` and other commands included.

Virtualbox's own [documentation](https://www.virtualbox.org/manual/ch08.html#vboxmanage-intro) also uses the `VBoxManage` command. 

Thanks,

**Comment by mebeim** [comment]
Thanks for the issue. It does indeed look like the command is misnamed. From what I can see from [the documentation](https://www.virtualbox.org/manual/ch08.html), at least.

Pinging @pxgamer since he created the page, maybe he knows something more. Let's wait for his response.

As for the second part: I agree, currently the page does not really offer much useful information apart from how to start a VM. If you're familiar with the tool a PR would be very welcome (after the naming of the tool is clear).

**Comment by pxgamer** [comment]
`virtualboxvm` is available on Windows. So maybe it is a Windows-only command. :thinking:

I think `vboxmanage` can be added to `common`, and the existing page can be moved to Windows. It was probably a mistake on my behalf thinking it worked on Linux as well.

**Comment by mebeim** [comment]
@pxgamer oh, so `virtualboxvm` only has that "start vm" option? Funny. I created #3069. Let me know if you happen to have any link to documentation about that tool that is more specific than Virtualbox's homepage, I couldn't find any.

**Comment by mebeim** [comment]
So, @danielmichaels looks like we don't have a `VBoxManage` page at all! Like I said, a PR to add it is more than welcome if you're familiar with the tool and would like to contribute. I see `VBoxManage` is a pretty complex tool.
